zookeeper-api/src/main/java/org/apache/helix/zookeeper/zkclient/ZkClient.java

```diff
@@ -204,6 +204,13 @@ public void recordPathStat(Stat stat, OptionalLong notificationTime) {
     }
   }
 
+  @Override
+  public String toString() {
```

Comment:
Same thing here, it seems that directly refer to _uid is enough.

Reply:
changed.